code/game/objects/items/devices/minimap_tablet.dm

```diff
@@ -0,0 +1,304 @@
+///A player needs to be unbanned from ALL these roles in order to be able to use the minimap drawing tool
+GLOBAL_LIST_INIT(roles_allowed_minimap_draw, list(SQUAD_LEADER, CAPTAIN, FIELD_COMMANDER, STAFF_OFFICER))
```

Comment:
The ability to draw like a 2 year old is now the most protected feature in the game. Truly befitting the unga.